### ISSUE TITLE
The "tasks" folder should be named "task"

### ISSUE THREAD

**Issue by hervelemeur** [jenkins-x/jx3-pipeline-catalog#70]
When running jx pipeline import --url https://github.com/jenkins-x/jx3-pipeline-catalog

I had this error:

> loading tekton tasks from: https://github.com/jenkins-x/jx3-pipeline-catalog ...

> error: the catalog does not include a 'task' directory at /var/folders/m4/ytp3wjnn1hlbrxt27d64jc3c0000gn/T/jx-git-658269897

**Comment by hervelemeur** [comment]
Here is the error I've got with `jx pipeline` and a custom catalog:

```
$ jx pipeline import --url https://github.com/myorg/jx3-pipeline-catalog
loading tekton tasks from: https://github.com/myorg/jx3-pipeline-catalog ...
error: the catalog does not include a 'task' directory at /var/folders/m4/ytp3wjnn1hlbrxt27d64jc3c0000gn/T/jx-git-300798298
```

**Comment by jstrachan** [comment]
yeah - FWIW we never assumed a jenkins x catalog would be usable as a tekton catalog ;) - but it makes sense we should detect them and play nicely

**Comment by jstrachan** [comment]
as we'd want to just generate `uses:jenkins-x/jx3-pipeline-catalog/...` steps now instead of the `kpt` approach

**Comment by hervelemeur** [comment]
Yeah, jsut re-read your comment on the PR... Should I rework it to actually rename the folder to `task` or do you prefer them to be better detected in the jx binaries? The former seems more in line with tekton.
